rascal-lsp/src/main/rascal/demo/lang/pico/LanguageServer.rsc

```diff
@@ -195,7 +195,7 @@ void main() {
             pathConfig(),
             "Pico",
             {"pico", "pico-new"},
-            "demo::lang::pico::NewLanguageServer",
+            "demo::lang::pico::LanguageServer",
```

Comment:
This fix should probably be released asap. I made a separate PR (#513) that targets `main` just for this.

Reply:
That PR has been merged so I have removed this change and merged changes from main into `error-recovery/rascal` and  subsequently `error-recovery/rascal` into `error-recovery/parametric`.